src/api/metadata.coffee

```diff
@@ -19,20 +19,20 @@ collections =
 
 
 #Function to remove properties from export object
-removeProperties = (obj) ->
+exports.removeProperties = (obj) ->
```

Comment:
Rather than export all of these and make them publicly available to other code, you should rather do something like this - https://github.com/jembi/openhim-core-js/blob/fc9413354df8a9df0377c517dede75d04868d20b/src/middleware/requestMatching.coffee#L131-L137